File: Python Programming/problems on file handling/problem1.py
'''
1. Write application which accept file name from user and open that file in read mode. 
	
	Input :  Demo.txt 
	
	Output :  File opened successfully.

	Author : Milind Nalawade

	Date : 27/08/2021

	Language : Python 3.9.0

'''
import os
from os import path
#=============
#Entry point function
#=============
def main():
	str=input("\n Enter file name : ")	
	
	if path.isfile(str):
		File=open(str,"r")
		print("\n File opened in read mode !!!\n")
	else:
		print("\n Unable to open the file  !!!\n")
	
#=====
#starter
#=====
if __name__=="__main__":
	main()

File: C++ Programming/problems on file handling/problem1.cpp
/*
1. Write application which accept file name from user and open that file in read mode. 
	
	Input :  Demo.txt 
	
	Output :  File opened successfully.

	Author : Milind Nalawade

	Date : 27/08/2021

	Language : C++ programming

*/
using namespace std;
#include<iostream>
#include<stdlib.h>
#include<unistd.h>
#include<fcntl.h>


//Entry point function
int main()
{
	char arr[20]={'\0'};
	int fd=0;
	
	cout<<"Enter file name : ";
	cin.getline(arr,20);
	fd=open(arr,O_RDONLY);
	if(fd==-1)
	{
		cout<<"\n Unable to open the file !!!\n";
	}
	else
	{
		cout<<"\n File open successfully !!!\n";
	}
	return 0;
}